### PR TITLE
Changes to .22, 9x19mm, 5.56 NATO and 7.62 NATO - V2

### PR DESCRIPTION
**Pull request by ghost**
Context: http://smf.cataclysmdda.com/index.php?topic=10693.0

Changes:
9x19mm FMJ replaced with 9x19mm JHP
Changed damage for .22LR, all 9x19mm rounds, 5.56 NATO and 7.62 NATO to √Ek
New names and descriptions for .22LR, 9x19mm, 5.56 NATO and 7.62 NATO

Recoil is currently unchanged because I have not yet worked out a consistent way to calculate it.

9mmjhp and 556 have minor arbitrary increases in damage to account for terminal ballistics.

I have made a second PR to make further work easier to keep track of.

### CODE REVIEW

**Comment by kevingranade** [comment]
You still have a bunch of unrelated changes bundled in here, look at the "files changed" tab.
Changing the id of an item will break saves as I outlined, the name change is fine, but the id needs to stay exactly as it is now.

**Comment by SpongeTheGlaive** [comment]
All instances of 9mmjhp have been changed back to 9mmfmj. What do you mean by unrelated changes?

File: data/json/item_groups.json [hunk 1]
```diff
@@ -1775,6 +1782,7 @@
       ["celery", 5],
       ["wheat", 3],
       ["onion", 3],
+      ["garlic", 3],
```
Comment (tvm1) [inline data/json/item_groups.json]: Changes like this.

File: data/json/item_groups.json [hunk 2]
```diff
@@ -8758,6 +8854,7 @@
     "id" : "loincloth",
     "items":[
       ["loincloth", 2],
+      ["loincloth_wool", 1],
```
Comment (tvm1) [inline data/json/item_groups.json]: Or this.

**Comment by SpongeTheGlaive** [comment]
I'll fix that.

**Comment by SpongeTheGlaive** [comment]
This is ready to go now. Further changes will go much smoother.

File: data/json/items/ammo.json [hunk 3]
```diff
@@ -803,11 +803,11 @@
     {   "type" : "AMMO",
         "id" : "9mmP2",
         "price" : 2700, "//" : "Same price as +P, but half the amount.",
-        "name" : "9mm +P+",
+        "name" : "9x19mm +P+",
         "name_plural" : "9mm +P+",
```
Comment (BevapDin) [inline data/json/items/ammo.json]: Plural is still only "9mm +P+" without the "x19" - intended?

**Comment by SpongeTheGlaive** [comment]
The lack of plural name changes was an oversight. The IDs are unchanged to prevent breaking save compatibility and to keep them short.

**Comment by DavidKeaton** [comment]
Minor note: Oxford comma is now standard, as is 2 spaces after a period. That's all! (proxy for Rivet, I'm possessed, send help)

**Comment by SpongeTheGlaive** [comment]
Serial commas and double spaces added. Restored inadvertently reverted changes to 556 and 762.

**Comment by DavidKeaton** [comment]
Righteous. :-) Thanks bud.

**Comment by chaosvolt** [comment]
Scheisse. Why do I gave the feeling that this is what hindered merging the 5.45x39mm mainlining? It would've gone in literally overnight if I hadn't missed a load error that had been present in Icecoon's mod for a while. :V

**Pull request merged**